### ISSUE TITLE
RNN LM decoding for Chinese corpus

### ISSUE THREAD

**Issue by Sundy1219** [srvk/eesen#147]
Now i don't want to use WFST decoder for Chinese ASR, is it possible to use RNN LM for decoding directly ? do you have any solutions ?Looking forward to your reply.Best wishes !!!

**Comment by fmetze** [comment]
Yes, the tf_clean branch has all the code, we are cleaning up a few things currently. @ramonsanabria  has more details

**Comment by Sundy1219** [comment]
Thanks for your reply.  But i am still confused to the problem that how to use RNNLM  to construct a decoder for Chinese ASR. 
 what is the acoustic modeling units ?  Chinese characters ?  such as   "你“  ”好"……？
can you give more information about it ?
Looking forward to your reply.Best wishes !!! @fmetze

**Comment by fmetze** [comment]
I have not tried an RNNLM with Chinese characters, but with a large enough training corpus, you should be able to directly use chinese characters, not? Isn’t this also what Baidu has done in their chinese deep speech version?
